dev/test/wdio.conf.js

```diff
@@ -141,7 +141,7 @@ exports.config = {
   mochaOpts: {
     ui: 'bdd',
     // debug hint: increase this timeout for debugging
-    timeout: 30003
+    timeout: 60006
```

Comment:
@smalls same as above -- checking that doubling here is ok since the comment above says it's to be increased only for debugging and this PR may have accidentally committed the change here and above.